### ISSUE TITLE
Yellorium ore removed from world

### ISSUE THREAD

**Issue by Dooglesmile** [ZeroNoRyouki/BigReactors#204]
Minecraft 1.12.2 version.
I have just updated Extreme Reactors in my mod pack from version 0.4.5.49 to 0.4.5.66, but on loading my world, I am no longer finding any Yellorium ore anywhere whilst mining, whereas before, I was finding it all over the place.

I did lose all the blocks and ingots in my AE2 system with the update. My own fault for not reading all the change logs for previous missed updates though. Luckily I'm only using the extreme reactor for backup power at the moment, so losing all sources of fuel isn't that big of a crisis just yet.

I've gone into the config files and set the B:enableWorldRegeneration=true and also I:userWorldGenVersion=1, but the ore doesn't seem to be regenerating as yet. I'm sat pretty much on the world spawn 0:0 and no ores found yet. 
Any idea what else I can try to get Yellorite ore to spawn again?

Going to new chunks to find ore is out of the question as the world was pregenerated to the world border!

**Comment by ZeroNoRyouki** [comment]
Did you run re-gen before? How did you pre-generated all the chunks? Could you paste me your ER config file on a nopaste service please?

**Comment by Dooglesmile** [comment]
Hi, I pregenerated the world using the Chunk Pregenerator mod. Took a while, but it was worth it to get rid of the lag caused by the other mods I'm using that generate structures etc. in the world.

I'm happy to say though, I was a little premature with my previous question about how else to get the ores to start respawning. I just needed to wait a little longer. They've started popping up again now :)

Sorry for being impatient, but thank you for making this mod in the first place. I love having my own little reactor to power stuff in the world :)
